src/store/index.js

```diff
@@ -3,9 +3,22 @@ import createPersistedState from "vuex-persistedstate";
 import plioItems from "./modules/plioItems";
 import auth from "./modules/auth";
 import sync from "./modules/sync";
+import SecureLS from "secure-ls";
+
+// encrypt and decrypt the localStorage
+// github.com/robinvdvleuten/vuex-persistedstate#encrypted-local-storage
+var ls = new SecureLS({ isCompression: false });
 
 export default createStore({
-  plugins: [createPersistedState()],
+  plugins: [
+    createPersistedState({
+      storage: {
+        getItem: (key) => ls.get(key),
+        setItem: (key, value) => ls.set(key, value),
+        removeItem: (key) => ls.remove(key),
```

Comment:
Better to switch to storing to and fetching from cookies instead of local Storage?